backend/src/routes/__tests__/sessions.test.js

```diff
@@ -0,0 +1,70 @@
+const {MongoMemoryServer} = require('mongodb-memory-server');
+const mongoose = require('mongoose');
+// const request = require('supertest');
+const sessions = require('../sessions');
+const express = require('express');
+
+jest.mock('../../index.js');
+
+let mongod;
+let server;
+let mockSession;
+
+const app = express();
+app.use(express.json());
+app.use('/sessions', sessions);
+
+beforeAll(async (done) => {
+  mongod = new MongoMemoryServer();
+
+  const connectionString = await mongod.getUri();
+  await mongoose.connect(connectionString, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+    useFindAndModify: false,
+  });
+
+  server = app.listen(0, () => {
+    port = server.address().port;
+    done();
+  });
+});
+
+beforeEach(async () => {
+  const coll = await mongoose.connection.db.createCollection('sessions');
+
+  mockSession = {
+    'preferences': {
+      'location': 'Auckland',
+      'distance': 10,
+      'cuisines': [
+        'Thai',
+        'Japanese',
+        'Chinese',
+      ],
+      'price': 20,
```

Comment:
I'm pretty sure this price is supposed to be between 1 and 4. Is this testing for a bug intentionally? if not please change to the accepted range